### PR TITLE
New documentation!

### PR DESCRIPTION
**Pull request by Dioscorides**
Rewrote the documentation, but it needs a little review / editing. Please give a look at the Markdown (.md) files in the /docs folder and provide feedback / edits.

1. Does the documentation make sense?
2. Does the documentation require further restructuring?
3. Any typos or weird pseudo-English sentences?

The scope of this review is the /docs folder only.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@
     Display your library's latest acquisitions using data coming from Alma.
     <br />
     <!--TODO-->
-    <a href="#"><strong>Explore the docs » (coming soon)</strong></a>
+    <a href="[#](https://acquisitions.library.universiteitleiden.nl/latest)"><strong>Explore the docs »</strong></a>
```
Comment (bencomp) [inline README.md:19]: You're making a Markdown link inside an href attribute?

File: README.md [hunk 2]
```diff
@@ -56,7 +56,7 @@
 
 [![Alma Acquisitions - Open Source][product-screenshot]](https://acquisitions.library.universiteitleiden.nl)
 
-**Alma Acquisitions - Open Source (AAOS)** is the open-source version of the [Leiden University Libraries - New Acquisitions](https://acquisitions.library.universiteitleiden.nl) app.
+**Alma Acquisitions - Open Source (AAOS)** is the open-source version of the [Acquisitions - New books at the library](https://acquisitions.library.universiteitleiden.nl) in use at Leiden University Libraries.
```
Comment (bencomp) [inline README.md:59]: maybe remove "the" before the link.

File: docs/introduction.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+**Alma Acquisitions - Open Source (AAOS)** is the open-source version of the [Acquisitions - New books at the library](https://acquisitions.library.universiteitleiden.nl) in use at Leiden University Libraries.
```
Comment (bencomp) [inline docs/introduction.md:1]: Again, remove "the" before the name of the application.

File: mkdocs.yml [hunk 4]
```diff
@@ -1,11 +1,11 @@
 ## Project information
-site_name: UBL Acquisitions
-site_url: https://acquisitions.library.universiteitleiden.nl/
-site_description: Display your library's most recent acquisitions!
+site_name: Alma Acquisitions - Open Source
+site_url: https://leidenuniversitylibrary.github.io/Alma-Acquisitions-Open-Source/
+site_description: "AAOS was created to display your library's latest acquisitions: e-books, journals, e-journals, etc. t was designed to allow the library's employees to manage the acquisitions lists via an clear and user-friendly admin interface."
```
Comment (bencomp) [inline mkdocs.yml:4]: Perhaps you could rephrase this to an active form. "AAOS displays ... allows employees ..."?

File: README.md [hunk 5]
```diff
@@ -73,7 +73,7 @@ AAOS was created to display the latest acquisitions made at Leiden University Li
 
 ## Getting Started
 
- Please refer to the [documentation](https://example.com) for more information about creating a test user, setting up the database, etc.
+ Please refer to the [documentation](https://acquisitions.library.universiteitleiden.nl/latest) for more information about creating a test user, setting up the database, etc.
```
Comment (bencomp) [inline README.md:76]: This link goes to the UBL's acquisitions website, not the documentation.

File: docs/alma-analytics.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+A --> |5. finally, sends data to | D[Alma Acquisitions Open Source];
+```
+
+!!! Important
+      **Before attempting to run this application and importing data, you must make sure that your Alma Analytics data configuration is properly setup.**
+
+!!! Note
+      Alma Analytics refreshes its data only once a day, furthermore it is build on top of Oracle Analytics Server (OAS)[^1]. For performance and usability reasons, this application stores and consumes data from a database, rather than relying on *on-the-fly* requests via the Alma Analytics API.
+
+### Creating a 'Latest Acquisitions' query in Alma Analytics
```
Comment (bencomp) [inline docs/alma-analytics.md:18]: I think the headers on this page should be `##`.

File: docs/creating-acquisitions-lists.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
+!!! NOTE
+    This part of the documentation assumes you have followed the instructions in [Part I - Alma Analytics](/Alma-Acquisitions-Open-Source/alma-analytics/) and [Part II - Installation](/Alma-Acquisitions-Open-Source/installation/)
```
Comment (bencomp) [inline docs/creating-acquisitions-lists.md:2]: I think you want to link to the .md files, as mkdocs will convert them and use the correct base URL.

File: docs/creating-acquisitions-lists.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+## Displaying the latest acquisitions
+
+!!! Important
+    Alma Acquisitions - Open Source expects the homepage to display the latest acquisitions; when a user visits the application, it will be redirected to the `/latest` page. This behavior can be edited in the `routes/web.php` file.
```
Comment (bencomp) [inline docs/creating-acquisitions-lists.md:7]: Use "they" to refer to a user, not "it".

File: docs/creating-acquisitions-lists.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+7. Click on `Create acquisitions list`
+
+If you navigate to the homepage, you should now see your library's latest acquisitions.
+
+!!! danger
+    You cannot delete the `latest` acquisitions list, but you can modify it. Be careful not to edit the `Acquisitions list's URL path` as AAOS always uses `latest` as the path to load the homepage.
+
+## Creating additional acquisitions lists
+
+Once you have created the `latest` acquisitions list, you are free to create as many acquisitions list you might need. First create the list in Alma Analytics [as explained in the documentation](Alma-Acquisitions-Open-Source/alma-analytics/#creating-an-acquisitions-list-for-a-specific-subject-in-alma-analytics), amd then follow steps 1 to 7 making sure to give a different name to your acquisitions list and URL path than `latest`.
```
Comment (bencomp) [inline docs/creating-acquisitions-lists.md:24]: Use .md files as link target.

File: docs/customizations.md [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+
+* `'primo_scope' => env('PRIMO_SCOPE', NULL),`
+The Primo Search Scope - Also known as [Search Profile](https://knowledge.exlibrisgroup.com/Primo/Product_Documentation/020Primo_VE/Primo_VE_(English)/040Search_Configurations/010Configuring_Search_Profiles_for_Primo_VE) in Primo VE - where your new acquisitions appear.
+
+* `'primo_vid' => env('PRIMO_VID', NULL),`
+The [Primo View Code](https://knowledge.exlibrisgroup.com/Primo/Product_Documentation/Primo/Back_Office_Guide/060Configuring_Primo%E2%80%99s_Front_End/020Views_Wizard) used at your institution.
+
+!!! NOTE
+      Alma Acquisitions (Open Source) searches for an item's MMS ID, which is a unique identifier; this is why in the URL construction we chose for `any,exact,MMS_ID`. If your institution prefers or requires another URL construction, you can easily edit this part to fit your needs by looking at the [Performing Advanced Searches](https://knowledge.exlibrisgroup.com/Primo/Product_Documentation/Primo/End_User_Help_-_New_UI/020Performing_Advanced_Searches) page in the documentation.
```
Comment (bencomp) [inline docs/customizations.md:47]: Don't say "easily": <https://justsimply.dev/>

File: docs/customizations.md [hunk 11]
```diff
@@ -0,0 +1,107 @@
...
+
+### Google Analytics
+
+This application can use Google Analytics to track usage if needed. You can configure your GA tracking ID in the `.env` file.
+
+Note that the IS called from the services file in `config/services.php`, like so:
```
Comment (bencomp) [inline docs/customizations.md:53]: "the IS called"?

File: docs/getting-started.md [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+## Start using your application
+
+On your local machine, you can use `php artisan serve` to launch a local version of your application. You will be able to login at <http://127.0.0.1:8000/login>
+
+If you have installed the application on your server, and used Tinker to create an admin account, proceed directly to `yourURL.com/login`.
+
+## Additional notes
+
+!!! Note
+    The main folder in Laravel is named `public`, some servers use `public_html` instead. Make sure your DNS points to the "public" folder.
```
Comment (bencomp) [inline docs/getting-started.md:60]: DNS points to IP addresses, the web server application (Apache HTTPD or nginx, for example) should be configured to have the correct directory as *document root*.

**Review by bencomp — COMMENTED** [review]
So I wasn't explicitly asked for feedback, but since I started before I knew that I figured I'd finish :)

Overall: great work! I think I might be able to install and configure the application based on these docs, even though I don't have Laravel experience and no Alma Analytics access.

I do wonder if there should be clearer distinction between things that all new users should configure (in environment variables) and more advanced customisation like changing the columns (in code)? Both are currently described in Customizations.

Besides the inline comments, I noticed that some file were deleted and others show as "empty". Or is that a GitHub glitch?

I saw quite some overlap in 'getting-started.md' and 'installation.md', but then saw that getting-started is not in the navigation.

File: README.md (hunk 1 above)
Comment (Dioscorides) [reply README.md:19]: This is actually two errors in one:

1. it should link to the github Page at https://leidenuniversitylibrary.github.io/Alma-Acquisitions-Open-Source , not to the app.
2. Not sure how md ended in there.
Well spotted!

File: README.md (hunk 5 above)
Comment (Dioscorides) [reply README.md:76]: Oepsie! That's what happens when one uses find and replace :D my bad!

File: docs/getting-started.md (hunk 12 above)
Comment (Dioscorides) [reply docs/getting-started.md:60]: Old habits, my bad. Changed to a more generic "Make sure your document root directory is the "public" folder."

File: mkdocs.yml (hunk 4 above)
Comment (Dioscorides) [reply mkdocs.yml:4]: Correct. I was being unnecessarily prosaic.

File: docs/creating-acquisitions-lists.md (hunk 7 above)
Comment (Dioscorides) [reply docs/creating-acquisitions-lists.md:2]: Didn't think about that at all! Thank you for the tip.

**Comment by Dioscorides** [comment]
@bencomp's corrections / comments have been added to the branch and will be pushed to the branch separately.
This Pull Request will be closed without merging.
